woocommerce/google-listings-and-ads
# Get started page UX improvements: add features card

### Changes proposed in this Pull Request:

Part of #1392, this PR adds the benefits card to the new get started page.

Design for desktop: eQ9O4m2flzrcAiDMXkOW0m-fi-7350%3A121114
Design for mobile: eQ9O4m2flzrcAiDMXkOW0m-fi-7476%3A122839

### Screenshots:

#### Desktop

<img width="1256" alt="Screenshot 2022-04-22 at 17 47 20" src="https://user-images.githubusercontent.com/914406/164682179-6bbcd6ab-9f62-461e-aeac-a474cd3dfd7b.png">

#### Mobile

<img width="1255" alt="Screenshot 2022-04-22 at 17 47 53" src="https://user-images.githubusercontent.com/914406/164682208-c630a979-8841-4d64-a7ff-d1f8e467fd87.png">

### Detailed test instructions:

1. Go to GLA's get started page: `https://domain.test/wp-admin/admin.php?page=wc-admin&path=%2Fgoogle%2Fstart`
2. Check the third card of the get started page, the UI should align with the design.

#### Testing for Tracking Events

1. Open your console and run `localStorage.setItem('debug', 'wc-admin*')`.
2. Enable verbose mode in dev tool's console.
3. Go to GLA's get started page: `https://domain.test/wp-admin/admin.php?page=wc-admin&path=%2Fgoogle%2Fstart`
4. Click on the first `Learn More`, it should open a new tab to https://woocommerce.com/document/google-listings-and-ads/#google-smart-shopping-campaigns. From the dev console you should see an event is sent with `link_id` = `get-started-features-free-listing-learn-more`.
5. Click on the second `Learn More`, it should open a new tab to https://woocommerce.com/document/google-listings-and-ads/#google-smart-shopping-campaigns. From the dev console you should see an event is sent with `link_id` = `get-started-features-google-ads-learn-more`.
6.  Click on the third `Learn More`, it should open a new tab to https://woocommerce.com/document/google-listings-and-ads/#getting-started-with-campaign-analytics. From the dev console you should see an event is sent with `link_id` = `get-started-features-dashboard-learn-more`.


### Changelog entry

> Update - Add the features card for UX improvements on get started page.
